src/eth_rpc/servers/debug_rpc.rs

```diff
@@ -219,4 +219,24 @@ impl<P: EthereumProvider + Send + Sync + 'static> DebugApiServer for DebugRpc<P>
 
         Ok(tracer.debug_transaction(transaction_hash)?)
     }
+
+    /// Runs an `eth_call` within the context of a given block execution and returns the Geth debug trace.
+    #[tracing::instrument(skip(self), err, fields(request = ?request, block_number = ?block_number, opts=?opts))]
```

Comment:
I think this will double log everything no?

Reply:
Yes that's true, I've just copy and paste the structure of the other functions of the same file but just figured out that we did the same mistake for the other ones, so just fixed everything. 

By the way, I don't think it double log but just useless and too verbose.